update tests for webpack 5 (#395)
GoogleChromeLabs/tooling.report: tests/non-js-resources/subtests/css/subtests/source/webpack/index.md

result: pass
---

> [css-loader] processes imported CSS files, which can then be passed through [extract-loader] to obtain the stylesheet text as a String. With webpack module type `asset/source` this string can be embedded as raw content in the bundle.

Reviewer: ```suggestion
[css-loader] processes imported CSS files, which can then be passed through [extract-loader] to obtain the stylesheet text as a string. Setting the module's `type` option to `"asset/source"` will embed this into the bundle.
```
